packages/module/patternfly-docs/content/extensions/quick-starts/examples/help-topics.md

```diff
@@ -4,22 +4,23 @@
 section: extensions
 # Sidenav secondary level section
 # should be the same for all markdown files for each extension
-id: Quick Starts
+id: Quick starts
 # Tab
-source: In-App-Documentation
+source: In\-app-documentation
```

Comment:
I think the "\" might be a typo?

Reply:
I was actually trying to escape the hyphen so that it would show up on the site as "In-app documentation" instead of "In app documentation"! It seemed like hyphens in the code weren't showing on the site, but I also wasn't sure if I understood how the webpage was building correctly. Do you know if that understanding is correct and if there's a way to make hyphens show?

Reply:
hi @jessiehuff wanted to loop back around on this question when you have a chance!